src/nvim/memory.c

```diff
@@ -496,7 +496,7 @@ void free_all_mem(void)
   block_autocmds();
 
   /* Close all tabs and windows.  Reset 'equalalways' to avoid redraws. */
-  p_ea = FALSE;
+  p_ea = false;
```

Comment:
When I pushed this commit I was unsure how to handle that. On a second, closer, examination, I think I should be able to fix this as you've suggested. I'll fix and re-push.